packages/documentation/docs/controls/_input_styleguide.md

```diff
@@ -18,17 +19,20 @@ An input field is a user interface element that allows users to enter and edit t
 - **Helper text:** See [form field](forms-field.md).
 - **Counter:** See [form field](forms-field.md).
 - **Feedback text**: See [form field](forms-field.md).
+
 ## Behavior in context
 
 - **Validation:** See [validation](forms-validation.md).
 - **Interaction**: Clicking in the container enables the editing of the field.
 - **Text truncation**: The text in an input field is cut off with the length of the container.
 - **Alignment**: Inputs are always aligned to the left, while right alignment is reserved exclusively for [number fields](number-input,md).
+
 ## States
 
-The input field has five states: default, focused, hover, disabled and read-only. In the read-only state, the input field is displayed without offering any user interaction.
+The input field has five states: default, active/focused, hover, disabled and read-only. In the read-only state, the input field is displayed without offering any user interaction.
+
+![Field States](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3198-7167&t=jhhv5OZGqmBpgXcs-4)
```

Comment:
[Select States] with link to https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3960-760&t=Yd1QtZiUYjzV5oPO-4